galleries.py

```
@@ -512,376 +537,179 @@ def _gallery(fileroot, product_ids, catalog, links,
+                        (url, title, _, alt) = subelement
+                    else:
+                        (url, title, alt) = subelement
+
+                    jump_to_links.append(create_menu_item(title, url, alt))
+
+
+        return jump_to_links
+
+    def create_floated_img_blocks(filename):
+        floated_img_blocks = []
+        for id in product_ids[filename]:
```

Ruff (0.15.9)

[error] Variable `id` is shadowing a Python builtin

(A001)